### PR TITLE
Update direct-routing-plan.md

### PR DESCRIPTION
**Pull request by NMuravlyannikov**

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-plan.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Users of Direct Routing must have the following licenses assigned in Office 365:
 
 - Microsoft Phone System 
 - Microsoft Teams + Skype for Business Plan 2 if included in Licensing Sku
-- Microsoft Audio Conferencing 
+- Microsoft Audio Conferencing (please read the notes and the paragraph below to see specific examples whed the license is requried)
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
- Microsoft Audio Conferencing (please read the notes and the paragraph below for specific examples and information about the license requirement)
```

File: Teams/direct-routing-plan.md [hunk 2]
```diff
@@ -92,10 +92,12 @@ Users of Direct Routing must have the following licenses assigned in Office 365:
 > [!IMPORTANT]
 >  In the case that you would like to add external participants to scheduled meetings, either by dialing out to them or by providing the dial-in number, the audio conferencing license is *required*.
 
-> [!NOTE]
-> The audio conferencing license is *required* to:
-> - Escalate from 1:1 call to a group call.
-> - Add external participants to scheduled meetings, by either dialing out or providing the dial-in number. 
+
+Ad Hoc call escalation and Audio Conferencing license.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
Ad hoc call escalation and Audio Conferencing license.
```

File: Teams/direct-routing-plan.md [hunk 3]
```diff
@@ -92,10 +92,12 @@ Users of Direct Routing must have the following licenses assigned in Office 365:
...
+
+A Teams User can start a one to one Teams to PSTN or Teams to Teams call and add a PSTN participant to it. This scenario called ad hoc conference. The path the call takes, depends if the Microsoft Audio Conferencing license assigned to the user who escalates the call or not.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
A Teams user can start a one on one Teams to PSTN or Teams to Teams call and add a PSTN participant to it. This scenario is called an ad hoc conference. The path that the call takes depends whether the Microsoft Audio Conferencing license assigned to the user escalates the call or not.
```

File: Teams/direct-routing-plan.md [hunk 4]
```diff
@@ -92,10 +92,12 @@ Users of Direct Routing must have the following licenses assigned in Office 365:
...
+1. If the Teams user, who escalates the call has the Microsoft Audio Conferencing license assigned, the escalation happens via Microsoft Audio Conferencing service. The remote PSTN participant who is being invited to the existing call on notification about incoming call sees the number of the Microsoft bridge, assigned to the Teams user who initiated the escalation
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
1. If the Teams user who escalates the call has a Microsoft Audio Conferencing license assigned, the escalation happens via the Microsoft Audio Conferencing service. The remote PSTN participant who is invited to the existing call receives a notification about the incoming call and sees the number of the Microsoft bridge assigned to the Teams user who initiated the escalation.
```

File: Teams/direct-routing-plan.md [hunk 5]
```diff
@@ -92,10 +92,12 @@ Users of Direct Routing must have the following licenses assigned in Office 365:
...
+2. If the Teams user, who escalates the call does not have the Microsoft Audio Conferencing license assigned, the escalation happens via a Session Border Controller, connected to Direct Routing interface. The remote PSTN participant who is being invited to the call on notification about incoming call sees the number of the Teams User who initiated the escalation. The specific SBC, used for the escalation, defined by Routing Policy of the user.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
2. If the Teams user who escalates the call does not have the Microsoft Audio Conferencing license assigned, the escalation happens via a Session Border Controller connected to the Direct Routing interface. The remote PSTN participant who is invited to the call receives a notification about the incoming call and sees the number of the Teams user who initiated the escalation. The specific SBC, used for the escalation, is defined by Routing Policy of the user. 
```

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**